apps/web/src/lib/helper/usage-formatter.ts

```diff
@@ -0,0 +1,34 @@
+import bytes from 'bytes';
+
+import { byteFormatter, numberFormatter } from '@cloudforet/core-lib';
+
+export const SIZE_UNIT_MAP = {
+    Bytes: 'B',
+    KB: 'KB',
+    MB: 'MB',
+    GB: 'GB',
+    PB: 'PB',
+    TB: 'TB',
+} as const;
+export type SizeUnitMap = keyof typeof SIZE_UNIT_MAP;
```

Comment:
suggestion: How about SizeUnit?? I remember that when defining types, I usually didn't append the "Map" postfix.

Reply:
I agree